packages/sui-i18n/src/adapters/polyglot.js

```diff
@@ -1,4 +1,222 @@
-import Polyglot from 'node-polyglot'
+// BSD 2-Clause License
+
+// Copyright (c) 2012, Airbnb
+// All rights reserved.
```

Comment:
ℹ️  license notice kept here as per:

https://github.com/airbnb/polyglot.js/blob/master/LICENSE

> 1. Redistributions of source code must retain the above copyright notice, this
>    list of conditions and the following disclaimer.

Reply:
👉 **A -personal- suggestion regarding this:** 

Would it make sense to have a specific file to maintain all third-party license notices? Relating each notice to the specific dependency that requires it.

Reply:
good point! not sure what's the best way to deal with this in our packages.

Reply:
We could open an issue to discuss this point and avoid blocking this PR. What do you think?

Reply:
totally agree! will do that then 👍